### ISSUE TITLE
Cannot add a third vocabulary backdrop 1.6

### ISSUE THREAD

**Issue by huizache** [backdrop/backdrop-issues#2505]
Fresh install, cannot add a third vocabulary in taxonomy. Page stays blank on page admin/structure/taxonomy/add
No dblog message

**Comment by olafgrabienski** [comment]
@huizache On my fresh install I was able to add a second and a third vocabulary to the existing vocabulary "Tags".

You write you wanted to add a third vocabulary. Does that mean that you was able to add a second one but not the third one?

**Comment by serundeputy** [comment]
@huizache I was also able to add a second and third vocabulary at `admin/structure/taxonomy/add` can you possibly provide the steps that led to the problem so we can try to replicate and solve?

thanks,
~Geoff

**Comment by huizache** [comment]
Yes I could add a second vocabulary. But could not add the third. Through _admin/structure/taxonomy/add_ I can write the name of the new vocabulary, description, etc but when I save page reloads to the same url _admin/structure/taxonomy/add_  but with a blank page

**Comment by huizache** [comment]
Ok, I have four sites with backdrop, all same code. Sites that only have english as language can add vocabulary. Sites that have a second language cannot add vocabulary. I remember now, I added the second vocabulary in the site opening this thread, before adding a second language.

**Comment by olafgrabienski** [comment]
@huizache Thanks for the feedback. I'll try to reproduce the issue. Some questions:

- What is the default language of the site?
- Have you configured *admin/config/regional/language/detection*, or are there still the default settings?

**Comment by huizache** [comment]
- Spanish
- Detection: default language
As mentioned above, going back to only english, as default, did not solve the problem

**Comment by olafgrabienski** [comment]
Ok, tested with German, *Detection and selection* set only to "Default language". Was able to add two vocabularies without problem.

I guess, German vs. Spanish shouldn't make a difference. No idea at the moment. I guess, we need more detailed steps to reproduce. Could you provide such a more detailed list? Like "Go to page Xyz (path)", click Button "A". You see "Something" (and so on)?

**Comment by jenlampton** [comment]
> Was able to add two vocabularies without problem.

@olafgrabienski but could you add three?

**Comment by olafgrabienski** [comment]
@jenlampton Yes, I could also three and even four vocabularies, so the number doesn't seem to be a problem per se.

**Comment by huizache** [comment]
Ok, there was a ghost role created, i do not know when or how (have a hypothesis), a role without name, without atributes, unable to delete it. I could see it in the role's edit page and in the add taxonomy vocabulary page. Seemed strange. I exported the config files looked for anything strange in taxonomy, and found the ghost role's config file. deleted it from the server. Now i can add new taxonomy vocabulary. This is in site that had all language modules uninstalled.

Added all the language modules, imported es.po, now it marks 99% translated, before it marked 51%. No extra role, I can create all taxonomy.

My guess is the first es.po importation messed up, and somehow created an extra role (ghost). This ghost role was blocking vocabulary creation.

I attach ghost role's json file with txt extension, and two screenshots of ghost role's presence in role edit page and taxonomy vocabulary creation:

![ghostrole1](https://cloud.githubusercontent.com/assets/107792/22578908/d954d1ce-e991-11e6-985f-f85537a9667a.png)
![ghostrole2](https://cloud.githubusercontent.com/assets/107792/22578919/ed828b32-e991-11e6-8ce7-0b31a6d078da.png)
[user.role..json.txt](https://github.com/backdrop/backdrop-issues/files/749543/user.role.json.txt)
